backend/dissemination/search.py

```diff
@@ -30,12 +48,18 @@ def search(params):
 
     ##############
     # GENERAL
-    results = search_general(params)
-    results = _sort_results(results, params)
-    results = search_alns(results, params)
-    results = search_findings(results, params)
-    results = search_direct_funding(results, params)
-    results = search_major_program(results, params)
+
+    if is_only_general_params(params):
+        results = search_general(DisseminationCombined, params)
```

Comment:
should this be

```python
results = search_general(General, params)
```

?

Reply:
I think, under `two`, we no longer need the branch between the two tables. And, in fact, I think the `if` can go away? @sambodeme and I saw this earlier, and wonder. I think this was a brief period of time where we thought we could improve things by keeping the general things on general... but, it doesn't matter for the use cases we're chasing?

Reply:
But... I'd be happy to pair and discuss. I'm not 100% certain. I *think* it wants to become `DisseminationCombined`, however. Because, otherwise, you're using a table that is "in the future" for some of your query, and a table that is "in the past" for other parts. 

Which is part of why we'll need to include a doc update (and the banner at the top of search) to say "search is updated once per hour," or similar. Someone coming to check if their audit was submitted won't know until the top of the hour (for example).

Reply:
yes that last point is what I had in mind when I asked -- not that querying `General` in this case would be better, but that it would always give up to date results. All things being equal, I'd prefer to not do that and have all of the search paths run against the same dataset, but wasn't sure how flexible that use case is, where people want to be extra sure that their submission really was accepted

Reply:
Unless you have a better idea, I think search has to live "one hour in the past."

We can, if we want, do several things:

1. Create a "is it submitted?" page. This is a UEI-only search against `general`. People can have an option to go to this page after they are done. (This could be a good solution, for all I know. While "single use," it addresses a point of stress that comes after submitting a big document of significance.)
2. We leave the site structure as-is, and document in multiple places that search updates once... per hour? Every 2 hours? (I don't know how long the refresh will take, or the load it will cause on the DB while rebuilding at this point.) We update the landing page where completed audits are shown to make clear "an audit listed here is considered complete and submitted to the FAC." 
3. ?

Either way, I agree search should run against one model/dataset only. I think we're currently in a world where we have to run against the `MATERIALIZED VIEW` and not the live models/dissemination tables.

Reply:
I'm essentially brainstorming in a GH thread. 

1. Users seem confused/unclear about their audit status on the internal landing page. We have struggled to make that text clear enough that they believe it represents a submitted audit.
2. I don't know if my #1 above is a use case. It's... a brainstorm in response to something you said, I think. Should we do it? I don't know. Possibly worth discussing, since it's a simple enough page... but it might not be worth the effort right now. Hence...

For the moment, I think we're in the category of #2: just use the MV. There's a delay for them to be able to verify their submission is in dissemination. :shrug: